README.md

```diff
@@ -1,65 +1,50 @@
 # Helm Hub
 
-The Helm Hub repository contains the documentation and some configuration for the
-distributed public repository search hosted by Helm that can be found at https://hub.helm.sh
+The Helm Hub repository contained the documentation and some configuration for the
+distributed public repository search hosted by Helm that used to be found at https://hub.helm.sh
 
-## Distributed Search
+## NOTICE: Redirect To Artifact Hub
```

Comment:
I've put these in various places for deprecation notices, and it seems to attract attention. WDYT?

```suggestion
## ⚠️ NOTICE: Redirect To Artifact Hub
```